### PR TITLE
Update top bar

### PR DESCRIPTION
**Pull request by rafalpawlow**
User / channel icons are not in scope
https://www.figma.com/file/Pf31tuYpozYmpq163U2ho8/Web-Components?node-id=3173%3A135755

### CODE REVIEW

File: src/components/_navigation/TopbarBase/TopbarBase.styles.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ export const Header = styled.header`
     calc(${sizes(4)} + 48px + ${sizes(2)});
 
   ${media.md} {
+    display: grid;
+    grid-template-rows: auto;
+    grid-template-columns: auto 480px 146px;
```
Comment (drillprop) [inline src/components/_navigation/TopbarBase/TopbarBase.styles.ts]: What if we change this to `1fr 480px 1fr`? I really don't like this 146px

File: src/components/_navigation/TopbarViewer/TopbarViewer.tsx [hunk 2]
```diff
@@ -81,21 +87,26 @@ export const TopbarViewer: React.FC = () => {
...
+            </Button>
+          ) : (
+            <Button icon={<SvgMember />} iconPlacement="left" size="medium" onClick={handleLogging}>
+              Sign up
+            </Button>
+          ))}
+        {!searchQuery && !mdMatch && !isLoggedIn && (
+          <>
+            <StyledIconButton onClick={handleLogging}>Sign up</StyledIconButton>
+          </>
```
Comment (drillprop) [inline src/components/_navigation/TopbarViewer/TopbarViewer.tsx]:
```suggestion
          <StyledIconButton onClick={handleLogging}>Sign up</StyledIconButton>
```
You don't need fragment here

**Review by drillprop — CHANGES_REQUESTED** [review]

**Review by drillprop — APPROVED** [review]
LGTM!

File: src/components/_icons/Bulb.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// THIS FILE WAS AUTOGENERATED BY SVGR. DO NOT MODIFY IT MANUALLY;
+import * as React from 'react'
+
+export const SvgBulb = React.memo((props: React.SVGProps<SVGSVGElement>) => (
```
Comment (kdembler) [inline src/components/_icons/Bulb.tsx]: The icons need to be re-generated because this uses some incorrect namings

File: package.json [hunk 4]
```diff
@@ -96,6 +96,7 @@
     "react-use-hotjar": "^1.2.0",
     "react-use-measure": "^2.1.0",
     "retry-axios": "^2.4.0",
+    "rimraf": "^3.0.2",
```
Comment (kdembler) [inline package.json]: Let's move this to dev dependencies

File: package.json [hunk 5]
```diff
@@ -32,7 +32,7 @@
     "svgs": "yarn svgs:icons && yarn svgs:illustrations",
     "svgs:illustrations": "svgr --config-file svgr.config.js -d src/components/_illustrations src/components/_illustrations/svgs && prettier --write src/components/_illustrations",
     "svgs:icons-import": "node ./scripts/figma-import",
-    "svgs:icons-generate": "svgr --config-file svgr.config.js -d src/components/_icons src/components/_icons/svgs && prettier --write src/components/_icons",
+    "svgs:icons-generate": "rimraf src/components/_icons/*.tsx && svgr --config-file svgr.config.js -d src/components/_icons src/components/_icons/svgs && prettier --write src/components/_icons",
```
Comment (kdembler) [inline package.json]: This removed `icons.stories` unfortunately

**Review by kdembler — APPROVED** [review]

**Pull request merged**